content/how-to/general/dns-fundamentals/index.md

```diff
@@ -491,34 +491,34 @@ whatever DNS server is the master for this zone.
   as **firstname@lastname.rackspace.com**.
 
 - **Serial number** - A 32-bit integer that indicates what version of the
-zone-file we are using. This is used by slave servers to determine if
-they have the same version of the zone file as the master server.
+zone-file we are using. This is used by secondary servers to determine if
```

Comment:
Weren't we using primary/replica? Why does this one start using "secondary" instead?

Reply:
because it's talking about timezone stuff not replicated databases.  there's a main server (the source of the time) and subservient or subsidiary servers that refer to the main one.  I struggled with whether to leave master and slave in place because the industry hasn't settled on an alternate yet. This was my compromise ...

Reply:
Gotcha